non_commercial_hosts.yaml

```diff
@@ -0,0 +1,365 @@
+---
```

Comment:
Is this file necessary? how is it different from hosts_example?

Reply:
This is definitely not necessary - I will remove it